### ISSUE TITLE
Update Aerospike Plugin To Include Extra Telemetry

### ISSUE THREAD

**Issue by grossjo** [influxdata/telegraf#7949]
## Feature Request

The aerospike input plugin is missing certain abilities.

### Proposal:

I wold like to add the following features, all optional features to the aerospike plugin

- output telemetry per set in a namespace
- output histograms on a per set and namespace 

### Current behavior:
N/A

### Desired behavior:
N/A

### Use case: 

Depending on your use case of aerospike, a lot of this telemetry gives valuable insight into operational costs of your database.

**Comment by ssoroka** [comment]
Hey Josh.

This sounds reasonable. Can you elaborate on whether you'd add new measurements or amend the existing ones? I'd love to see example metrics. What data would be in the histogram?

Cheers,
Steven ;)

**Comment by grossjo** [comment]
It will be adding additional measurements.
One each for "sets", "ttl", "object-size". 

I'm still working out what these will look like once parsed into telegraf format 

The sets measurement will work very similar to the existing one in the plugin:
ns=test:set=demo:objects=12345:tombstones=0:memory_data_bytes=851805:truncate_lut=0:stop-writes-count=0:\
set-enable-xdr=use-default:disable-eviction=false

I'm still sorting out how to convert a histogram to something that works in the telegraf world.

**Comment by ssoroka** [comment]
sets, ttl, and object-size sound like they should be properties of a new or existing metric. Not super familiar with aerospike at the moment. 
TL;DR on the influx line protocol is `metric_name,tag1=str1,tag2=str2 field1=val1,field2=val2 {time.Now().UnixNano()}`. Unlike prometheus, this typically means metric_name is a resource, and not a property of a resource.

You can look at the [histogram aggregator](https://github.com/influxdata/telegraf/blob/master/plugins/aggregators/histogram/README.md#histogram-aggregator-plugin) for examples on how to work with histograms in Telegraf/InfluxDB. The accumulator object exposed to plugins has a AddHistogram() function that sets the field value type, but I think its use is optional, and it should work regardless (I've not tested this).

**Comment by grossjo** [comment]
I've opened up a PR that has the changes: https://github.com/influxdata/telegraf/pull/8025
